themes/bootstrap3/js/vendor/jquery-ui.min.js

```diff
@@ -0,0 +1,6 @@
+/*! jQuery UI - v1.13.2 - 2023-02-20
```

Comment:
How critical is jQuery UI to this PR? We are currently in the early stages of removing all jQuery dependencies from the codebase, so I'm reluctant to add a major new dependency if it can be avoided...

Reply:
I think it is only used to enable a tab for each configured language in the edit forms for pages and broadcasts. The reordering of entries in the list view to change the displayed order is done with the DataTables library.

Is it just a problem using jQuery UI particularly, so that I can try and find a smaller, independent library for the tabs? Or should additional JavaScript libraries be avoided?

Reply:
I'm currently trying to refactor the notifications language-tabs to the way tabs are implemented on the record page.

Reply:
I did not get the tabs from the detail view to work in the notifications context, so I build a small JS/CSS tab navigation myself. jQuery UI is removed.

The less and sass files are included. Should all the generated files be committed too?

```
/themes/bootprint3/css/compiled.css
/themes/bootstrap3/css/compiled.css
/themes/bootstrap3/css/vendor/simple-keyboard/index.css
/themes/bootstrap3/js/vendor/chart.js
/themes/bootstrap3/js/vendor/simple-keyboard/index.js
/themes/bootstrap3/js/vendor/simple-keyboard-layouts/index.js
/themes/local_theme_example/css/compiled.css
/themes/sandal/css/compiled.css
```

Reply:
Thanks, @jschultze, and yes, these generated files should be committed. (We commit all artifacts that are generated using Node.js-based tools, so that it is not a hard requirement to have Node installed to run or develop VuFind).

Reply:
Looks like we don't have the simple-keyboard version pinned, so it's getting updated too.

Reply:
I guess the question is whether we should pin our versions, or if we should just be more careful about using `npm [install/update] [package-name]` to avoid updating extraneous things.

Reply:
The files are included now.

Reply:
@demiankatz `composer update-npm-dependencies` (or `npm run updateDeps` for a single theme) doesn't allow you to do it for a single package and can cause side-effects unless you're careful.

Reply:
Good point, @EreMaijala. I guess we need to decide on a strategy here. I think there are two general approaches:

1.) We pin all dependencies, and add composer/npm tasks to help identify pinned dependencies that are eligible for upgrades.

2.) We leave dependencies unpinned, and I open a separate PR to upgrade and test everything, and then we merge that separately to remove extraneous changes from this PR.

Any preference?

Reply:
For now, I've opened #3522 to bring things up to date independently of this PR... but I'm still open to discussing long term strategy independent of that. :-)

Reply:
@demiankatz I don't have a strong preference, but pinning would be consistent with composer dependencies and would avoid the situation here where just trying to add or update one dependency will easily cause side-effects.

Reply:
It occurs to me that the best solution might be to start committing the lock file. Can you think of any reason not to?